app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from datetime import date
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Vaccine is not")
+        elif visitor["vaccine"]["expiration_date"] < date.today():
+            raise OutdatedVaccineError("Vaccine is outdated")
+        elif not visitor["wearing_a_mask"]:
```

Comment:
It would be better to check it with the get method, as 'wearing_a_mask' might be absent in the visitor dictionary